invenio_search_ui/assets/js/invenio_react_search_ui/Results.js

```diff
@@ -0,0 +1,134 @@
+// /*
+//  * This file is part of Invenio.
+//  * Copyright (C) 2020 CERN.
+//  *
+//  * Invenio is free software; you can redistribute it and/or modify it
+//  * under the terms of the MIT License; see LICENSE file for more details.
+//  */
+
+import React, { Component } from "react";
+import { Grid, Card, Item, Label } from "semantic-ui-react";
+import _truncate from "lodash/truncate";
+import _get from "lodash/get";
+import {
+  Count,
+  LayoutSwitcher,
+  Pagination,
+  ResultsMultiLayout,
+  ResultsList,
+  ResultsGrid,
+  Sort
+} from "react-searchkit";
+import PropTypes from "prop-types";
+
+export class Results extends Component {
+  constructor(props) {
+    super(props);
+
+    this.sortValues = this.props.sortValues;
+  }
+
+  renderResultsListItem = (result, index) => {
+    const metadata = result.metadata;
+    return (
+      <Item key={index} href={_get(result, "links.html", "#")}>
```

Comment:
I solved the link like this: https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/226
(also removed `truncate` and change style... do we actually need `lodash`)?

Reply:
I don't see the solution for the `links.html` under your link. Could you explain in more details, which line do you mean?